src/style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  color: rgba(6, 11, 53, 1);
+}
+
+body {
+  margin: 0;
+}
+
+:root {
+  --color: rgba(97, 96, 112, 1);
+  --button-color: rgba(0, 172, 220, 1);
+  --hover-button-color: rgba(255, 255, 255, 1);
+}
+
+.card {
+  padding: 32px 0 16px;
+  width: 198px;
+  background: var(--hover-button-color);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__img {
+  margin: 0 19px 40px;
+}
+
+.card-img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin: 0 16px 4px;
+}
+
+.name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__code {
```

Comment:
should be in one row
<img width="146" alt="Screenshot 2023-04-05 at 21 11 35" src="https://user-images.githubusercontent.com/117587463/230167838-b6954066-0571-4b76-9c6a-66bead49f31b.png">